packages/service/db/migrations/20220717162732_document-history-access-policy.sql

```diff
@@ -0,0 +1,22 @@
+-- migrate:up
+ALTER TABLE document_history ENABLE ROW LEVEL SECURITY;
+CREATE POLICY select_document_history_if_allowed ON document_history
+  FOR SELECT
+  TO postgraphile_user
+  USING (
+    document_id IN (
+        select id
+        from document
+        where document.id = document_id
```

Comment:
this is basically saying that if you can select this document on the document table, then you can select from this table also.

Selection from the document table is controlled by another policy
https://github.com/chewnoill/collaborative-editor/blob/f5da3a59d157176285e14134872f2a8b7a30c95e/packages/service/db/migrations/20210827184910_public-links.sql#L6-L17